### ISSUE TITLE
As a WOAQ team member, I want to understand our approach to accessibility

### ISSUE THREAD

**Issue by theecrit** [openoakland/woeip#66]
**Is your feature request related to a problem? Please describe.**
When building a community-based tool intended to serve underrepresented populations, a11y becomes a fundamental issue of equity. 

**Describe the solution you'd like**
A documented set of design principles that define our approach to and process for building an accessible solution. It should outline:
* Core principles.
* What standards/best practices we'll implement (I'd like to see us shoot for WCAG 2.1 AA, with fallback baseline when that's not possible).
* A guide or checklist for implementing specific requirements within any given Issue/design that respects the iterative approach to our work.

**Describe alternatives you've considered**
Foundation's out-of-the-box "accessibility" is known to place the onus on the programmer, resulting in code that often doesn't meet best practice standards.

**Additional context**
While HTML and css implementation cover many a11y issues, best practices go well beyond that and impact UX and content, as well as code structure. Section 508 requires WCAG 2.0 compliance. However, CA law also requires compliance with “Web Content Accessibility Guidelines 2.0, or a subsequent version” (at the AA level)[source](https://www.dor.ca.gov/Home/AB434). WCAG 2.1 was released mid-2018 and is additive to 2.0, to account for many digital realities not originally accounted for in 2.0. All of it is in service of four core principles: making the product *perceivable*, *operable*, *understandable*, and *robust*.

Because this may be prohibitive for an MVP launch, it’s worth proactively coming up with a plan to reach some baseline standards. I’m looking into whether C4A has any specific standards, but in the meantime there are lots of great resources out there to help us.

### Technical Resources
* [WCAG quick reference guide](https://www.w3.org/WAI/WCAG21/quickref/)
* [Accessibility for Teams from UK.gov](https://accessibility.digital.gov/)
* [Web accessibility evaluation tool](https://wave.webaim.org/)
* [Foundation Accessibility Introduction](https://foundation.zurb.com/sites/docs/accessibility.html)

**Comment by theecrit** [comment]
@TangoYankee Does it make more sense to move this into Trello as a Project Management card? Then the output of this process (a set of guidelines/checklist) can be attached to new implementation Issues.

**Comment by TangoYankee** [comment]
@theecrit yes. It would be better managed in a Trello Card. Also, I added a link to the Web accessibility tool that I've used in the past and Zurb's introduction to the topic.

**Comment by theecrit** [comment]
Closing; moved to [Trello](https://trello.com/c/zfsYqv5H).
